### PR TITLE
Document badges, cron, contexts and podspec api

### PR DESCRIPTION
**Pull request by fejta**
Document: 

* How to create a badge.
* Where to get info about what a podspec is
* Note how to `override` contexts
* Required components/plugins

### CODE REVIEW

File: prow/jobs.md [hunk 1]
```diff
@@ -244,5 +264,18 @@ Batch Job:
 
 See ["How to test a ProwJob"](/prow/build_test_update.md#How-to-test-a-ProwJob).
 
-[`Presets`]: https://github.com/kubernetes/test-infra/blob/3afb608d28630b99e49e09dd101a96c201268739/prow/config/jobs.go#L33-L40
+## Badges
+
+Prow can display badges that signal whether jobs are passing ([example](https://prow.k8s.io/badge.svg?jobs=post-test-infra-bazel)).
+
+The format to send your `deck` URL is `/badge.svg?jobs=single-job-name` or `/badge.svg?jobs=common-job-prefix-*`.
+
```
Comment (michelle192837) [inline prow/jobs.md]: Concrete example?

File: prow/jobs.md [hunk 2]
```diff
@@ -156,17 +172,21 @@ to Prow in some other context, like `/close`. It is similarly not suggested to d
...
+Presubmit and postsubmit jobs post a status context to the GitHub
+commit under test once they start, unless the job is configured
+with `skip_report: true`.
+
+Use a `/retest` or `/test job-name` to re-trigger the test and
+hopefully update the failed context to passing.
+
+If a job should no longer trigger on the pull request, use the
+`/skip` command to dismiss a failing status context (depends on
+`skip` plugin).
```
Comment (michelle192837) [inline prow/jobs.md]: Maybe a link to the skip plugin docs, or similar? (Same for override below)

File: prow/jobs.md (hunk 1 above)
Comment (fejta) [reply prow/jobs.md]: Isn't example in the previous paragraph concrete?

Comment (michelle192837) [reply prow/jobs.md]: ...Yup it is! ><

**Comment by fejta** [comment]
/hold cancel

Will send more improvements in a follow-up

**Pull request merged**